system/modules/core/pages/PageRegular.php

```diff
@@ -87,6 +89,7 @@ public function generate($objPage, $blnCheckRequest=false)
 			{
 				while ($objModules->next())
```

Comment:
If we use `foreach` on the collection, you could use `@var \ModuleModel[] $objModules`